Gemfile

```
@@ -130,7 +130,7 @@ group :test do
 end
 
 platforms :ruby, :mswin, :mswin64, :mingw, :x64_mingw do
-  gem "nokogiri", ">= 1.8.1", "!= 1.11.0"
+  gem "nokogiri", ">= 1.13.2", "!= 1.11.0"
```

Nit:
If we still think this change make sense, I believe we could get rid of the `!= 1.11.0` part as `>= 1.13.2` already excludes this version by implication
```suggestion
  gem "nokogiri", ">= 1.13.2"
```

---

Ah my bad, this PR https://github.com/rails/rails/pull/44579 merged 4 weeks back has updated it to 1.13.x. But I guess it's good to be explicit about the version.